presto-hive/src/main/java/com/facebook/presto/hive/statistics/MetastoreHiveStatisticsProvider.java

```diff
@@ -87,16 +89,20 @@ public MetastoreHiveStatisticsProvider(TypeManager typeManager, SemiTransactiona
     }
 
     @Override
-    public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTableHandle tableHandle, List<HivePartition> hivePartitions, Map<String, ColumnHandle> tableColumns)
+    public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTableHandle tableHandle, List<HivePartition> selectedPartitions, Map<String, ColumnHandle> tableColumns)
```

Comment:
I do find `selectedPartitions` to be a better name than `hivePartitions`. I would refrain from this rename. Maybe `allTablePartitions`?

Reply:
Surely `don't find`. Yes `allTablePartitions` might be not accurate. Maybe `allQueriedPartitions`?

Reply:
Let it be `allQueriedPartitions`